snuba/settings/__init__.py

```diff
@@ -58,6 +58,7 @@
 )
 
 ADMIN_ALLOWED_PROD_PROJECTS: Sequence[int] = []
+ADMIN_ALLOWED_ORG_IDS: Sequence[int] = []
```

Comment:
do you know how this is supposed to be populated?

Reply:
its gotta be an ops repo change...